### PR TITLE
Switched from using pixels to meters

### PR DESCRIPTION
**Pull request by hej2010**
Bodies now collide with each other

### CODE REVIEW

File: android/AndroidManifest.xml [hunk 1]
```diff
@@ -4,6 +4,8 @@
     android:versionCode="1"
     android:versionName="1.0" >
 
+    <uses-feature android:glEsVersion="0x00020000" android:required="true" />
```
Comment (EmElw) [inline android/AndroidManifest.xml]: What does this do?

File: core/src/com/pottda/game/controller/KeyboardMouseController.java [hunk 2]
```diff
@@ -22,8 +22,8 @@ public KeyboardMouseController(ModelActor modelActor, ViewActor viewActor, Stage
 
     @Override
     public void onNewFrame() {
-        movementVector.set((Gdx.input.isKeyPressed(Input.Keys.D) ? 1 : 0 - (Gdx.input.isKeyPressed(Input.Keys.A) ? 1 : 0)) * 120, // 120 for testing
-                (Gdx.input.isKeyPressed(Input.Keys.W) ? 1 : 0 - (Gdx.input.isKeyPressed(Input.Keys.S) ? 1 : 0)) * 120);
+        movementVector.set((Gdx.input.isKeyPressed(Input.Keys.D) ? 1 : 0 - (Gdx.input.isKeyPressed(Input.Keys.A) ? 1 : 0)) * 3,
+                (Gdx.input.isKeyPressed(Input.Keys.W) ? 1 : 0 - (Gdx.input.isKeyPressed(Input.Keys.S) ? 1 : 0)) * 3);
```
Comment (EmElw) [inline core/src/com/pottda/game/controller/KeyboardMouseController.java]: 3 looks like an odd number, should maybe be a final constant

File: core/src/com/pottda/game/model/Character.java [hunk 3]
```diff
@@ -60,6 +60,14 @@ private void attack(Vector2f direction) {
         inventory.attack(direction, getPosition());
     }
 
+    @Override
+    public float getAngle() {
+        if (team == 1) { // Fix rotation for enemies so they rotate towards player
```
Comment (EmElw) [inline core/src/com/pottda/game/model/Character.java]: Looks like a sketchy solution, but lower priority

**Review by EmElw — DISMISSED** [review]

File: core/src/com/pottda/game/controller/KeyboardMouseController.java (hunk 2 above)
Comment (LatteG) [reply core/src/com/pottda/game/controller/KeyboardMouseController.java]: Any constant should be a final static <T>

File: core/src/com/pottda/game/model/Character.java (hunk 3 above)
Comment (LatteG) [reply core/src/com/pottda/game/model/Character.java]: 1 should be replaced by a final static int with the value 1

**Review by EmElw — APPROVED** [review]

File: core/src/com/pottda/game/MyGame.java [hunk 4]
```diff
@@ -100,13 +103,13 @@ public void create() {
...
         controllers.add(ActorFactory.get().buildPlayer(gameStage,
-                new Texture(Gdx.files.internal(playerImage)), new Vector2f(gameStage.getWidth() / 2, gameStage.getHeight() / 2)));
+                new Texture(Gdx.files.internal(playerImage)), new Vector2f(WIDTH_METERS / 2, HEIGHT_METERS / 2)));
 
         // Add some enemies
         for (int i = 0; i < 5; i++) {
             try {
                 controllers.add(ActorFactory.get().buildEnemy(gameStage, new Texture(Gdx.files.internal(enemyImage)), //Change
-                        new Vector2f((float) Math.random() * gameStage.getWidth(), (float) Math.random() * gameStage.getHeight()),
+                        new Vector2f((float) (4 + Math.random() * (WIDTH_METERS - 4)), (float) (4 + Math.random() * (HEIGHT_METERS - 4))),
```
Comment (LatteG) [inline core/src/com/pottda/game/MyGame.java:112]: What is this 4? It should be a constant with a descriptive name.

**Pull request merged**